monero-outreach-docs/translations/pt-br/PSA/files/forks/forks_pt-br.md

```diff
@@ -1,12 +1,12 @@
 # Conheça seus hard forks
 
-A não ser que você seja desenvolvedor, o termo "hard fork" pode causar muita confusão e potencialmente trazer conotações negativas para você. Afinal, você já deve ter lido um artigo ou outro sobre hard forks e como ele causou contenção ou drama em alguns projetos de cripto. Os hard forks do Monero acontecem de maneira interessante e enquadram-se em três categorias principais:
+A não ser que você seja desenvolvedor, o termo "hard fork" pode causar muita confusão e eventualmente trazer conotações negativas. Afinal, você já deve ter lido um ou outro artigo sobre hard forks e como eles causaram divisões ou conflitos em alguns projetos de cripto. Os hard forks do Monero acontecem de maneira interessante e enquadram-se em três categorias principais:
 
 ### 1) Atualização da rede / Hard fork regularmente programado
 
-Os desenvolvedores do Monero têm por objetivo realizar dois hard forks regulares por ano. Esses hard forks são sempre feitos com o consenso da comunidade e liderado pelos Desenvolvedores Core. Eles geralmente incluem diversas melhorias ao código principal do Monero. É necessário que você, como usuário Monero, atualize suas carteiras sempre que isso ocorrer e que não é responsabilidade de ninguém, senão você mesmo, de fazê-lo. Ninguém disse que seria fácil ser um pioneiro da tecnologia, mas se for reparar bem, não é tão difícil também.
```

Comment:
I haven't seen this one: "liderado" on the second line must be in plural: "liderados"

Reply:
Good catch. Thanks again @botoserelepe